nntrainer/opencl/opencl_program.cpp

```diff
@@ -21,13 +21,20 @@
 
 #include <nntrainer_log.h>
 
+#ifdef OPENCL_KERNEL_PATH
+const std::string DEFAULT_KERNEL_PATH = OPENCL_KERNEL_PATH;
+#else
+const std::string DEFAULT_KERNEL_PATH = "";
```

Comment:
In the next PR (you may go ahead with this PR and make another PR to fix it), please enable envvar or .ini file configuration for DEFAULT_KERNEL_PATH. Having the default file search path hardcoded is not recommended although it's fine if it can be overriden by envvar or .ini file.

And... it'd be more proper if it's a static-const property of the Program class.